core/templates/directives/mathjax-bind.directive.ts

```diff
@@ -33,7 +33,10 @@ angular.module('oppia').directive('mathjaxBind', [function() {
             ).html(value === undefined ? '' : value);
             $element.html('');
             $element.append($script);
-            MathJax.Hub.Queue(['Reprocess', MathJax.Hub, $element[0]]);
+            // @ts-ignore
```

Comment:
Add a comment for why this needs to be added (see other places in the codebase for examples).

Reply:
removed this